com.unity.ml-agents/CHANGELOG.md

```diff
@@ -72,6 +72,8 @@ and this project adheres to
 
 ### Minor Changes
 
+- Updated the Dockerfile and renamed it to ml-agents.Dockerfile. The docker image
```

Comment:
Since this isn't going in the 1.00/0.16.0 release, please add it to the `[Unreleased]` section above.